src/Uno.Toolkit.UI/Controls/ExtendedSplashScreen/ExtendedSplashScreen.xaml

```diff
@@ -5,7 +5,7 @@
 					xmlns:utu="using:Uno.Toolkit.UI"
 					mc:Ignorable="d">
 
-	<x:String x:Key="DefaultLoadingViewAnimationDuration">00:00:00.083</x:String>
+	<x:String x:Key="DefaultExtendedSplashScreenAnimationDuration">00:00:00.083</x:String>
```

Comment:
If you want to make this customizable through something like [Lightweight Styling](https://learn.microsoft.com/en-us/windows/apps/design/style/xaml-styles#lightweight-styling), we can define this in ThemeDictionaries and instead use the `ThemeResource` binding in the VisualStates. So people could override this value without needing to redefine the style.

Sort of like here: https://github.com/unoplatform/uno/blob/9d9619f2af3b5b86b8c0f03a8ee9e5538d2b88f7/src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/SplitView_themeresources.xaml#L7-L41

Reply:
done